amp/WEB-INF/src/org/digijava/module/aim/util/ActivityVersionUtil.java

```diff
@@ -70,14 +71,20 @@ public static boolean implementsVersionable(Class[] array) {
         return ret;
     }
 
-    public static String generateFormattedOutput(HttpServletRequest request, Output out) throws WorkerException {
-        return generateFormattedOutput(request, out, null);
+    //public static String generateFormattedOutput(HttpServletRequest request, Output out) throws WorkerException {
```

Comment:
same as before, when we are changing a method is difficult to trace down what has changed with commented code, if you see what has changed in this pr you can notice. So the best practice is to just change the code that needs to be changed, and if you need to